haiku-apps/2dphysicsdemo/2dphysicsdemo-22.5.98.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="2D Physics demo"
+DESCRIPTION="2D Physics Test Program is a cheesy test harness for 2D physics."
+HOMEPAGE="https://github.com/HaikuArchives/2dPhysicsDemo"
+COPYRIGHT="1997 Chris Hecker
+	Andreas Kaenner"
+LICENSE="unknown"
```

Comment:
Reminder: If we keep `LICENSE="unknon"` then the package won't be available in the depot.

Well, I guess we have no choice...

Reply:
There is this in the sourcecode:

This material is Copyright 1997 Chris Hecker, All Rights Reserved.
It's for you to read and learn from, not to put in your own articles
or books or on your website, etc.  Thank you.

That looks like a licence to me. It does not give us much rights however. We can now contact Chrish Hecker (http://chrishecker.com/Rigid_body_dynamics) and ask for something more permissive, I guess.

Personally I won't miss this much in the depot, anyways. It's nice as a programming sample, but not so much useful as a packaged app?